package.json

```diff
@@ -44,13 +44,19 @@
   ],
   "scripts": {
     "toc": "doctoc README.md",
-    "lint": "(prettier . --check || true) && eslint .",
+    "lint": "prettier . --check && eslint .",
```

Comment:
If "unformatted files exist" is an expected state of this repository, then `lint` shouldn't error if there are unformatted files; I think the `|| true` should be added back to `prettier` to downgrade it back to warnings